sync/windows/forked.ps1

```diff
@@ -1,12 +0,0 @@
-Param(
```

Comment:
This file is used by Vagrant, as you've mentioned in your PR description. It shouldn't be deleted.